### PR TITLE
Query vendor_loads db for vendors list

### PR DESCRIPTION
**Pull request by lwrubel**
Completes part of #332 to list vendors from the database. Tests are not included here, will be ticketed separately.

### CODE REVIEW

File: libsys_airflow/plugins/vendor_app/vendors.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+from flask_appbuilder import expose, BaseView as AppBuilderBaseView
+from folioclient import FolioClient
 import requests
+from sqlalchemy.orm import Session
 
 from airflow.models import Variable
-
-from flask_appbuilder import expose, BaseView as AppBuilderBaseView
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from plugins.vendor.models import Vendor
```
[the remark this answers is not in the document]
Comment (lwrubel) [reply libsys_airflow/plugins/vendor_app/vendors.py:11]: I re-ordered and re-grouped the imports to follow PEP-8 https://peps.python.org/pep-0008/

[the remark this answers is not in the document]
Comment (ndushay) [reply libsys_airflow/plugins/vendor_app/vendors.py:11]: I'm thinking along these lines, but again, I'm not sure of python convention:

```
import logging
import os
import requests

from airflow.models import Variable
from airflow.providers.postgres.hooks.postgres import PostgresHook
from flask_appbuilder import expose, BaseView as AppBuilderBaseView
from folioclient import FolioClient
from plugins.vendor.models import Vendor
from sqlalchemy.orm import Session
```

does that make my concern clearer?

**Review by ndushay — CHANGES_REQUESTED** [review]

**Comment by lwrubel** [comment]
I realized I wasn't totally following PEP-8 and further adjusted the ordering to follow that. How's this look now? 

```
import logging
import os

from airflow.models import Variable
from airflow.providers.postgres.hooks.postgres import PostgresHook
from flask_appbuilder import expose, BaseView as AppBuilderBaseView
from folioclient import FolioClient
import requests
from sqlalchemy.orm import Session

from plugins.vendor.models import Vendor
```

**Comment by lwrubel** [comment]
I've rebased to remove the merge conflict. I won't be available to take any changes further in this PR in the coming week while I'm away, so if there are further edits needed, would someone else be able to take them? Thanks!

**Review by edsu — APPROVED** [review]

**Comment by ndushay** [comment]
@edsu if you want to resolve conflicts and fix the imports and then merge this, go for it.

**Review by ndushay — APPROVED** [review]

**Pull request merged**